### ISSUE TITLE
Need base indentation value

### ISSUE THREAD

**Issue by tacshooter** [sasstools/sass-lint#1280]
Simple issue - there appears to be no setting or configuration for a base indentation value. Therefore, you get an error/warning reported when the first line has an indentation. This is the case when the sass has been indented from a script tag, which is certainly not what I would prefer in our code. I would propose a configuration similar to vue/script-indent, where you set a baseIndent value which works as a multiplier. When set to 0, it allows no indentation at the beginning. However, when set to 1, it requires that first line to be indented from the script tag. The default could be 0, so as to work exactly as it currently does without this setting.

**Comment by tacshooter** [comment]
This SHOULD NOT produce an error or a warning when you indent the sass contained in the style tag.
```
<style scoped lang="scss">
  div.someclass {
    margin: 0 20px 21px 0;
    max-width: 450px;
  }
</style>
```
